homeassistant/components/device_tracker/owntracks.py

```diff
@@ -182,6 +186,23 @@ def leave_event():
                 data['event'])
             return
 
+    def owntracks_waypoint_update(topic, payload, qos):
+        """List of waypoints published by a user."""
+        # Docs on available data:
+        # http://owntracks.org/booklet/tech/json/#_typewaypoints
+        data = validate_payload(payload, 'waypoints')
+        if not data:
+            return
+
+        wayps = data['waypoints']
+        _LOGGER.info("Got %d waypoints from %s", len(wayps), topic)
+        for wayp in wayps:
+            name = wayp['desc']
+            lat = wayp['lat']
```

Comment:
There are 3 places were we are using `'lat'` and `'long'` please use constants - if there are other magic strings please convert those as well